tests/integration_tests/functional/test_logging.py

```diff
@@ -146,6 +148,8 @@ def test_api_requests_logs(test_microvm_with_api):
         show_level=True,
         show_log_origin=True,
     )
+    # only works if log level is Debug
+    microvm.time_api_requests = False
```

Comment:
Should we move this then to the microvm constructor? No point in timing if we know it's not going to work...